feedback2go/Views/CorrectionSidebar/AddFeedbackView.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  FeedbackCell.swift
+//  feedback2go
+//
+//  Created by Katjana Kosic on 18.11.22.
+//
+
+import Foundation
+import SwiftUI
+
+struct AddFeedbackView: View {
```

Comment:
size is buggy and interferes with the assessment view

Reply:
fixed